src/java.base/share/classes/sun/util/locale/provider/JavaTimeDateTimePatternImpl.java

```diff
@@ -63,10 +66,24 @@ public boolean isSupportedLocale(Locale locale) {
     @Override
     public String getJavaTimeDateTimePattern(int timeStyle, int dateStyle, String calType, Locale locale) {
         LocaleResources lr = LocaleProviderAdapter.getResourceBundleBased().getLocaleResources(locale);
-        String pattern = lr.getJavaTimeDateTimePattern(
+        return lr.getJavaTimeDateTimePattern(
                 timeStyle, dateStyle, calType);
-        return pattern;
+    }
 
+    @Override
+    public String getJavaTimeDateTimePattern(String requestedTemplate, String calType, Locale locale) {
+        LocaleProviderAdapter lpa = LocaleProviderAdapter.getResourceBundleBased();
+        // CLDR's 'u'/'U' are not supported in the JDK. Replace them with 'y' instead
+        final var modifiedSkeleton = requestedTemplate.replaceAll("[uU]", "y");
```

Comment:
This was a left over which should be removed. In fact, CLDR specific pattern symbols should not be accepted as the requested template. Removed the substitiutions. As to the validation, it will be performed in the following `LocaleResources.getLocalizedPattern()` method.